homeassistant/components/ihc/ihcdevice.py

```diff
@@ -41,17 +66,27 @@ def name(self):
         """Return the device name."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return f"{self.controller_id}-{self.ihc_id}"
+
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
-        if not self.info:
+        if not self.hass.data[DOMAIN][self.controller_id][CONF_INFO]:
             return {}
-        return {
+        attributes = {
             "ihc_id": self.ihc_id,
             "ihc_name": self.ihc_name,
             "ihc_note": self.ihc_note,
             "ihc_position": self.ihc_position,
         }
+        multicontroller: bool = len(self.hass.data[DOMAIN]) > 1
```

Comment:
We don't need a variable. There's a comment below which is enough.
```suggestion
        if len(self.hass.data[DOMAIN]) > 1:
```